shared/js/ui/models/whitelist.es6.js

```diff
@@ -46,9 +46,10 @@ Whitelist.prototype = window.$.extend({},
             // But first, strip the 'www.' part, otherwise getSubDomain will include it
             // and whitelisting won't work for that site
             url = url ? url.replace('www.', '') : ''
```

Comment:
since we are here - can you review and merge https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/428/files ?